cmd/configs.go

```diff
@@ -80,6 +80,9 @@ func lintConfig(fileType configFileType, configFile string) *gerr.GatewayDError
 		if err := conf.LoadGlobalConfigFile(context.TODO()); err != nil {
 			return err
 		}
+		if err := conf.ConvertKeysToLowercase(context.TODO()); err != nil {
```

Comment:
Not directly related to your PR, but I see a context.TODO here

I'm surprised by its usage. This one differs from context.Background because it expects the developers to fix it at some point.

The lintConfig method could/should be updated to pass the context, even it's one context.TODO in another method, it would be better than having 6+ context.TODO in this method.

If it's too much work for now, the lintConfig method could be unchanged but a `ctx := context.TODO()` variable could be added and reused everywhere in this method. Because right now, the code is very strange to read

Of course, this could be addressed outside this PR

Reply:
Any thoughts you woul like to share about this ?

@sinadarbouy

Reply:
Yes, I agree that we should not have `context.TODO()` in non-test files. In a separate PR, we should handle context with at least basic context handling for all contexts.

Reply:
Thanks